pkg/trtl/metrics/metrics.go

```diff
@@ -132,36 +133,42 @@ func initMetrics() {
 		Namespace: PmNamespace,
 		Name:      "sync_latency",
 		Help:      "total duration of anti-entropy (originator perspective), labeled by peer and region",
+		Buckets:   prometheus.LinearBuckets(100, 200, 50),
 	}, []string{"peer", "region"})
 
 	PmAEPhase1Latency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
 		Namespace: PmNamespace,
 		Name:      "phase1_latency",
 		Help:      "duration of anti-entropy phase 1 (originator perspective), labeled by peer",
+		Buckets:   prometheus.LinearBuckets(100, 200, 50),
```

Comment:
I recommend starting the bins at 1 for both p1 and p2 latency and having a smaller step

```golang
prometheus.LinearBuckets(1, 10, 50)
```